app/src/main/res/layout/fragment_habit_event_view.xml

```diff
@@ -0,0 +1,104 @@
+        android:layout_height="200dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/habit_temp_img" />
+
+    <TextView
+        android:id="@+id/view_event_date_label"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="81dp"
+        android:background="@color/habit_add_button"
+        android:text="@string/view_event_date_label"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        android:textSize="24sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/imageView2"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/view_event_comment_label"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="@color/habit_add_button"
+        android:text="@string/view_event_comment_label"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        android:textSize="24sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView2" />
+
+    <TextView
+        android:id="@+id/view_event_location_label"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:background="@color/habit_add_button"
+        android:text="@string/view_event_location_label"
+        android:textAlignment="center"
+        android:textColor="@color/white"
+        android:textSize="24sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/view_event_comment" />
+
+    <TextView
+        android:id="@+id/view_event_comment"
+        android:layout_width="0dp"
+        android:layout_height="200dp"
+        android:background="@color/white"
+        android:inputType="textMultiLine"
+        android:singleLine="false"
+        android:text="@string/view_event_comment_label"
+        android:textAlignment="textStart"
+        android:textColor="@color/theme_dark_grey"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/view_event_comment_label" />
+
+    <TextView
+        android:id="@+id/temp_location_placeholder"
+        android:layout_width="0dp"
+        android:layout_height="200dp"
+        android:background="#C4C4C4"
+        android:gravity="center"
+        android:inputType="textMultiLine"
+        android:singleLine="false"
+        android:text="GOOGLE MAPS SNIPPET WITH LOCATION HERE"
```

Comment:
I'm assuming the reason behind why this isn't extracted into a string resource is because it's temporary, in which case this is fine

Reply:
Correcto